en/installation/installation-of-a-central-server/using-packages.md

```diff
@@ -73,123 +73,48 @@ systemctl disable firewalld
 ### Install the repositories
 
 <!--DOCUSAURUS_CODE_TABS-->
-<!--RHEL 8-->
-#### Redhat CodeReady Builder repository
-
-To install Centreon you will need to enable the official CodeReady Builder
-repository supported by Redhat.
-
-Enable the CodeReady Builder repository using these commands:
-
-```shell
-dnf -y install dnf-plugins-core https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm
-subscription-manager repos --enable codeready-builder-for-rhel-8-x86_64-rpms
-```
-
-Enable PHP 7.3 using the following command:
-```shell
-dnf module enable php:7.3 -y
-```
-
-Check that PHP 7.3 is activated:
-```shell
-dnf module list php
-```
-
-You should have this result:
-```shell
-Red Hat Enterprise Linux 8 for x86_64 - AppStream (RPMs)
-Name                                     Stream                                 Profiles                                                 Summary
-php                                      7.2 [d]                                common [d], devel, minimal                               PHP scripting language
-php                                      7.3 [e]                                common [d], devel, minimal                               PHP scripting language
-php                                      7.4                                    common [d], devel, minimal                               PHP scripting language
-
-Hint: [d]efault, [e]nabled, [x]disabled, [i]nstalled
-```
-<!--CentOS 8-->
-#### Redhat PowerTools repository
-
-To install Centreon you will need to enable the official PowerTools repository
-supported by Redhat.
-
-Enable the PowerTools repository using these commands:
-
-- For Centos 8.2:
-
-    ```shell
-    dnf -y install dnf-plugins-core epel-release
-    dnf config-manager --set-enabled PowerTools
-    ```
+<!--CentOS 7-->
```

Comment:
@kevin, the CentOS7 tab is in second after the other repos in the installation documentation, may you fix the display order please ?